src/flux/course.coffee

```diff
@@ -46,7 +47,7 @@ CourseConfig =
     @emit('practice.loaded', obj.id)
 
   _reset: ->
-    CrudConfig.reset.call(@)
+    CrudConfig.reset.call(this)
```

Comment:
is explicitly saying `this` preferred over using `@` "standalone"/in these types of changes?